android/lib/src/main/java/com/airbnb/android/react/maps/AirMapMarkerManager.java

```diff
@@ -23,11 +22,7 @@
     public static final int SHOW_INFO_WINDOW = 1;
     public static final int HIDE_INFO_WINDOW = 2;
 
-    private AirMapCalloutManager calloutManager;
-
-    public AirMapMarkerManager(AirMapCalloutManager calloutManager) {
-        super();
-        this.calloutManager = calloutManager;
```

Comment:
strange. wonder if this is a bug we didnt know about, or just an oversight? lol. I must have put it there for a reason???